packages/bundler-plugin-core/src/types.ts

```diff
@@ -278,6 +278,17 @@ export interface Options {
     excludeReplayWorker?: boolean;
   };
 
+  /**
+   * Options related to react component name annotations.
+   * Disabled by default, unless a value is set for this option
+   */
+  reactAnnotate?: {
```

Comment:
Let's make this option name a bit more detailed

```suggestion
  reactComponentNameAnnotate?: {
```